include/revng/Yield/Support/SugiyamaStyleGraphLayout.h

```diff
@@ -16,11 +16,22 @@ enum class RankingStrategy {
   DisjointDepthFirstSearch
 };
 
+enum class LayoutOrientation {
+  LeftToRight,
+  RightToLeft,
+  TopToBottom,
+  BottomToTop
+};
+
 struct Configuration {
 public:
   RankingStrategy Ranking;
+  LayoutOrientation Orientation;
   bool UseOrthogonalBends;
 
+  bool PreserveLinearSegments;
+  float VirtualNodeWeight;
+
   Graph::Dimension NodeMarginSize;
   Graph::Dimension EdgeMarginSize;
 };
```

Comment:
I didn't do so already because I'm waiting for the pipeline's global options support - after that 
is ready, these would need to be moved on top of that anyway, so I'd rather wait with documentation until that point.

Reply:
Uhm, I'd prefer to have documentation now, then we can move it. It makes reviewing the code much easier.

Reply:
Sure.

Reply:
Is this what you have in mind? https://github.com/revng/revng/pull/191/commits/8ba22a1fa050eddadce907f18ab68b6e6001b38d